### PR TITLE
Added Centered Square Number

### LINKED ISSUE #1370: Centered Square Number in C, C++, Java, Python and go

### PR DESCRIPTION
**Pull request by somya-kapoor**
Fixes #1370

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Centered Square Number in C, C++, Java, Python and go

#### Languages Used:
- C
- C++
- java
- go 
- python

#### Files Added:
- Centered_Square_Number.c
- Centered_Square_Number.cpp
- Centered_Square_Number.java
- Centered_Square_Number.go
- Centered_Square_Number.py

### CODE REVIEW

File: Centered_Square_Number/Centered_Square_Number.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+    Centered Square Number is a centered figurate number that gives the number
+    of dots in a square with a dot in the center and all other dots surrounding 
+    the center dot in successive square layers.
+
+    Nth Centered square number can be calculated by using formula n^2 + (n-1)^2.
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.c]: No need to use indentation inside comment here.

File: Centered_Square_Number/Centered_Square_Number.c [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+*/
+
+#include<stdio.h> 
+
+int centeredSquare (int num)
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.c]: Remove space after ``centeredSquare``.

File: Centered_Square_Number/Centered_Square_Number.cpp [hunk 3]
```diff
@@ -0,0 +1,32 @@
+/*
+    Centered Square Number is a centered figurate number that gives the number
+    of dots in a square with a dot in the center and all other dots surrounding 
+    the center dot in successive square layers.
+    
+    Nth Centered square number can be calculated by using formula n^2 + (n-1)^2.
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.cpp]: Here as well, no need for internal indentation.

File: Centered_Square_Number/Centered_Square_Number.cpp [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+*/
+
+#include<iostream> 
+using namespace std; 
+   
+
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.cpp]: Remove extra line.

File: Centered_Square_Number/Centered_Square_Number.go [hunk 5]
```diff
@@ -0,0 +1,30 @@
+/*
+    Centered Square Number is a centered figurate number that gives the number
+    of dots in a square with a dot in the center and all other dots surrounding 
+    the center dot in successive square layers.
+    
+    Nth Centered square number can be calculated by using formula n^2 + (n-1)^2.
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.go]: Fix here as well.

File: Centered_Square_Number/Centered_Square_Number.go [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+*/
+
+package main
+
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.go]: Remove extra line.

File: Centered_Square_Number/Centered_Square_Number.go [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+func centeredSquare (num int) int {
+    // Using formula 
+    return num * num + ((num - 1) * (num - 1))
+}
+
+func main() {
+    var num int
+    fmt.Scanf("%d", &num)
+
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.go]: I guess there's no need for an extra blank line here...

File: Centered_Square_Number/Centered_Square_Number.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
+/*
+    Centered Square Number is a centered figurate number that gives the number
+    of dots in a square with a dot in the center and all other dots surrounding 
+    the center dot in successive square layers.
+    
+    Nth Centered square number can be calculated by using formula n^2 + (n-1)^2.
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.java]: Fix

File: Centered_Square_Number/Centered_Square_Number.java [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    public static void main(String[] args)  
+    {     
+        int num;
+        Scanner sc = new Scanner(System.in);
+        num = sc.nextInt();
+        System.out.println(centeredSquare(num)); 
+    } 
+} 
+
+
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.java]: Remove one extra line.

File: Centered_Square_Number/Centered_Square_Number.py [hunk 10]
```diff
@@ -0,0 +1,22 @@
+'''
+    Centered Square Number is a centered figurate number that gives the number
+    of dots in a square with a dot in the center and all other dots surrounding 
+    the center dot in successive square layers.
+    
+    Nth Centered square number can be calculated by using formula n^2 + (n-1)^2.
```
Comment (marwahmanbir) [inline Centered_Square_Number/Centered_Square_Number.py]: Fix

**Comment by somya-kapoor** [comment]
@marwahmanbir Done, please review

**Pull request merged**